### DISCUSSION TITLE
如何在K8S中配置mysql数据库连接配置？Halo.2.7

### DISCUSSION THREAD

**Discussion by chenyuncai** [halo-dev/halo#4244] [Q&A]
当前看官网docker-compose文件中，配置外部数据库是通过command命令来设置，但是在k8s中，这几个变量是如何工作的？为何不是普通的env定义环境变量？

当前可部署
1. 重新编写Dockerfile, 替换镜像中的application.yaml方式，然后部署新的镜像。
2. 采用docker-compose单独部署
期望:
数据库相关配置采用环境变量优先读取的方式。

**Comment by JohnNiang** [comment]
请参考文档：<https://docs.halo.run/getting-started/install/helm>。
